src/main/scala/is/hail/HailContext.scala

```diff
@@ -164,7 +164,8 @@ object HailContext {
     ProgressBarBuilder.build(sparkContext)
 
     val sqlContext = new org.apache.spark.sql.SQLContext(sparkContext)
-    val hc = new HailContext(sparkContext, sqlContext, tmpDir, branchingFactor)
+    val hcTemp = TempDir.createTempDir(tmpDir, sparkContext.hadoopConfiguration)
```

Comment:
Can you change this variable name? I think hc* is confusing because it's not a HailContext.

Reply:
Will do.